### PR TITLE
add class to target js after links and not display

### PR DESCRIPTION
**Pull request by Juliet-Brown**
Fix [DS-365](https://govtnz.atlassian.net/jira/software/projects/DS/boards/87?selectedIssue=DS-365) . Make after links not show when` js ` is on.

### CODE REVIEW

File: packages/govtnz-ds-website/src/commons/styles/elements-global.scss [hunk 1]
```diff
@@ -109,6 +109,15 @@ body {
     display: inline;
   }
 
+  // so JS users can't see the after links
+  .no-after-link-js {
+    body {
```
Comment (holloway) [inline packages/govtnz-ds-website/src/commons/styles/elements-global.scss:114]: Is the `body` part of this irrelevant?

File: packages/govtnz-ds-website/src/components/layout.tsx [hunk 2]
```diff
@@ -26,6 +26,10 @@ const Layout = (props: Props) => {
       document.documentElement.classList.add('theme-default', 'js');
     }
 
+    if (typeof window !== 'undefined') {
```
Comment (holloway) [inline packages/govtnz-ds-website/src/components/layout.tsx:29]: Why are we adding the `js` class twice, on line 26 and line 30?

Comment (Juliet-Brown) [reply packages/govtnz-ds-website/src/components/layout.tsx:29]: removed, we don't need it here.

File: packages/govtnz-ds-website/src/commons/styles/elements-global.scss (hunk 1 above)
Comment (Juliet-Brown) [reply packages/govtnz-ds-website/src/commons/styles/elements-global.scss]: No I don't  think so I was going off the code underneath which uses body but it doesn't need it. Have removed it

File: packages/govtnz-ds-website/src/components/layout.tsx (hunk 2 above)
Comment (holloway) [reply packages/govtnz-ds-website/src/components/layout.tsx:29]: Thanks.. the preceding code checks for whether `document.body.classList` exists before using it but this code doesn't. Perhaps we should move this code into that `if` block on line 26?

File: packages/govtnz-ds-website/src/components/layout.tsx [hunk 3]
```diff
@@ -23,7 +23,11 @@ const Layout = (props: Props) => {
 
   useEffect(() => {
     if (document.body.classList) {
-      document.documentElement.classList.add('theme-default', 'js');
+      document.documentElement.classList.add(
```
Comment (holloway) [inline packages/govtnz-ds-website/src/components/layout.tsx:26]: This needs the `typeof window !== 'undefined'` check restored

**Review by holloway — APPROVED** [review]

**Pull request merged**